dynamiq/evaluations/metrics/answer_correctness.py

```diff
@@ -0,0 +1,267 @@
+import json
```

Comment:
@delamainer would these be supported on the backend now?

Reply:
Probably not these RAGAS-like complex metrics yet, but we will add their support to catalyst as soon as we merge them into dynamiq